src/platform/datapath_raw_xdp.c

```diff
@@ -837,22 +846,32 @@ CxPlatDpRawInitialize(
                 }
 
                 CxPlatZeroMemory(Interface, sizeof(*Interface));
+
+                Interface->IfIndex = Adapter->IfIndex;
                 memcpy(
                     Interface->PhysicalAddress, Adapter->PhysicalAddress,
                     sizeof(Interface->PhysicalAddress));
+                Interface->OffloadStatus.Receive.NetworkLayerXsum = Xdp->SkipXsum;
+                Interface->OffloadStatus.Receive.TransportLayerXsum = Xdp->SkipXsum;
+                Interface->OffloadStatus.Transmit.NetworkLayerXsum = Xdp->SkipXsum;
+                Interface->OffloadStatus.Transmit.NetworkLayerXsum = Xdp->SkipXsum;
+                CxPlatListInitializeHead(&Interface->Flows);
 
-                Status =
-                    CxPlatDpRawInterfaceInitialize(
-                        Xdp, Interface, Adapter->IfIndex, ClientRecvContextLength);
+                Status = CxPlatGetInterfaceRssQueueCount(Interface->IfIndex, &Interface->RssQueueCount);
                 if (QUIC_FAILED(Status)) {
-                    QuicTraceEvent(
-                        LibraryErrorStatus,
-                        "[ lib] ERROR, %u, %s.",
-                        Status,
-                        "CxPlatDpRawInterfaceInitialize");
                     CxPlatFree(Interface, IF_TAG);
                     continue;
                 }
+
+                // TODO - Don't hard code flows. Dynamically create them in response
+                // to sockets getting created. Will be done in follow up PR.
```

Comment:
what type of sockets are you referring to? XSKs or CXPLAT_SOCKETs?

Reply:
Eventually, they will be equivalent, but right now, XSKs.